### PR TITLE
init docs site

### PR DESCRIPTION
**Pull request by Ethan-Arrowood**
First stab at generating a docs site from the markdown documents. Docsify seems cool and I like the DX so far. I'll get a github actions thing working soon so it "auto deploys" on new commits and such

Check out the live preview: http://ethanarrowood.com/undici/#/

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# undici
```
Comment (mcollina) [inline docs/README.md:1]: This file seems a duplication of the project README. is this correct? Can we avoid that duplication?

Comment (Ethan-Arrowood) [reply docs/README.md:1]: Yeah this is an unfortunate part of using these site generators. They sorta need everything to be included in the same directory. We could write some sort of copy script thats included in the build pipeline. From my cursory review of other options (docusaurus and vuepress), docsify is the simplest

Comment (mcollina) [reply docs/README.md:1]: Check out https://github.com/mercurius-js/mercurius configuration. It does not duplicate the README.

Comment (Ethan-Arrowood) [reply docs/README.md:1]: oh smart - but then the site file lives in repo root. If we are cool with that I'll do it here too

Comment (mcollina) [reply docs/README.md:1]: Better than duplicating!

**Comment by mcollina** [comment]
That would be amazing.

**Comment by Ethan-Arrowood** [comment]
Yes I'd love to make that all happen together. I'll try and send Agent docs ASAP as well as wrap up this site deployment work

**Comment by Ethan-Arrowood** [comment]
We could also use a logo 😉

**Review by dnlup — APPROVED** [review]
This looks amazing!

**Comment by ronag** [comment]
Why does CI fail?

**Comment by mcollina** [comment]
It passes now!

**Pull request merged**